int16 debug_num
int16[100] sensor_data_x
int16[100] sensor_data_y
bool op3_local_mode
int16 start_point_x
int16 start_point_y
int16 start_point_orien
